movo_desktop/movo_viz/launch/view_robot.launch

```diff
@@ -3,6 +3,13 @@
 <launch>
   <arg name='function' default=''/>
 
+  <include file="$(find movo_description)/launch/movo_upload.launch"/>
+
+  <include file="$(find movo_moveit_config)/launch/move_group.launch">
```

Comment:
its a little strange for the view_robot launch file to be launching move groups. isnt the point of this launch file to just view the robot?

Reply:
Yeah that's what I thought but when I try to run the view_robot.launch file it raises an error about not having the movo description and after I add the upload launch file it states that I need the semantic description. Did you not see these when you ran it?

Reply:
hmm why does move_group need to be launched with view_robot.launch? did u just try adding different launch files until it worked? i dont think move_group should be launched with view_robot

Reply:
![view_robot_err](https://user-images.githubusercontent.com/42899188/195730381-8745d996-8e83-4370-ab6e-25f7a5f386b4.png)
Yeah I tried to resolve the errors I found when running the launch file with roslaunch. I added the movo_upload to get the robot description and then the movo_group launch to fix a semantic description error. If this runs on master fine then perhaps I'm missing a change and I can remove the lines